index.html

```diff
@@ -0,0 +1,1546 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <!-- Required meta tags -->
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <script src="https://kit.fontawesome.com/c7d73dbfbb.js" crossorigin="anonymous"></script>
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous" />
+  <link rel="stylesheet" href="./css/style.css" />
+
+  <title>Newsweek Clone</title>
+</head>
+
+<body>
+  <nav class="nav-1 container-fluid">
+    <div class="row">
+      <div class="col-lg-4 d-none d-lg-block d-xl-block">
+        <div class="my-nav-weather">
+          <img src="30.png" alt="" class="weather" />
+          <span class="first-span">30, Belgrade, RS &nbsp;<i class="fas fa-chevron-right"></i></span>
+          <br />
+          <span class="second-span">Wed, Sep 16, 2020</span>
+        </div>
+      </div>
+      <div class="col-lg-4 col-4 d-xl-block d-lg-block">
+        <div class="my-nav-1">
+          <img src="./images/images.png" alt="Responsive image" class="h-100 w-100" />
+        </div>
+      </div>
+      <div class="col-lg-4 col-8 nav-btns row align-items-center justify-content-end">
+        <button type="submit" class="d-none d-lg-block btn-login">
+          LOGIN</button>&nbsp;&nbsp;
+        <i class="d-lg-none fas fa-user-alt"></i>
+        <button type="button" class="btn btn-primary btn-sm bg-warning border-0 nav-subscribe mx-3">
+          SUBSCRIBE >
+        </button><i class="d-lg-none d-xl-none fas fa-bars"></i>
+      </div>
+    </div>
+  </nav>
+  <!--  -->
```

Comment:
- [ ] Please delete **commented piece of codes** in your HTML and CSS files. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */